.gitignore

```diff
@@ -10,4 +10,5 @@ user.bazelrc
 # XCHammer specific things to ignore for now
 external
 *-XCHammer*.xcodeproj
-xchammer-includes
\ No newline at end of file
+xchammer-includes
+xcbuildkit-data
```

Comment:
The downside of this `xcbuildkit-data` living in the workspace root is everyone has to now add a `.gitignore` for it / see more stuff at the root. `xchammer-includes` was 100% necessary to put there unfortunately

Reply:
We can also put this under `path/to/foo.xcodeproj/xcbuildkit-data`. Thoughts?

Reply:
https://github.com/bazel-ios/rules_ios/pull/577#discussion_r1014397768